src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.xml

```diff
@@ -18,27 +20,27 @@
                 </argument>
             </call>
         </service>
-        <service id="Doctrine\Common\Annotations\Reader" alias="annotations.reader" public="false" />
+        <service id="Doctrine\Common\Annotations\Reader" alias="annotations.reader" />
```

Comment:
@nicolas-grekas this autowiring alias should point to `annotation_reader`, which is the public name for it (even though it is an alias), so that people decorating the public name still get it applied in case of autowiring.